packages/gasket-plugin-metadata/lib/actions.js

```diff
@@ -62,7 +63,20 @@ async function getMetadata(gasket) {
         pluginData.metadata.path = path.join(app.metadata.path, 'plugins');
         plugins.push(pluginData);
       } else {
-        pluginData.metadata.path = path.dirname(path.join(require.resolve(pluginData.name), '..'));
+        let resolvedPath;
+        try {
+          resolvedPath = require.resolve(pluginData.name, { paths: [gasket.config.root] });
```

Comment:
`{ paths: [gasket.config.root] }` allow any unhoisted dependencies to also get resolved. in a monorepo if plugin-metadata is a common it dependency and hoisted at the root and another dependency which uses the metadata hook but is only used by a single package in the monorepo this was throwing an error. starting the resolution at gasket app root fixes this